### PR TITLE
Add L_LU_REJECTION_REASON lookup table

### PR DESCRIPTION
**Pull request by clark800**

### CODE REVIEW

File: src/models/db/transfers.js [hunk 1]
```diff
@@ -31,6 +36,11 @@ function convertFromPersistent (data) {
   if (data.rejected_at) {
     data.rejected_at = new Date(data.rejected_at)
   }
+  if (!isNil(data.rejection_reason_id)) {
```
Comment (MatthewPhinney) [inline src/models/db/transfers.js]: Is this just a safety check? Or do some dbs return `undefined` and others return `null`?

File: src/sql/pg/create.sql [hunk 2]
```diff
@@ -24,14 +24,33 @@ CREATE INDEX "L_XIE_FINGERPRINT" ON "L_ACCOUNTS"
...
+CREATE TABLE IF NOT EXISTS "L_LU_REJECTION_REASON" (
+  "REJECTION_REASON_ID"   INTEGER NOT NULL,
+  "NAME"                  CHARACTER VARYING(10) NOT NULL,
+  "DESCRIPTION"           CHARACTER VARYING(255) NULL
+);
+
+CREATE INDEX "L_XPK_LU_TRANSFERS_REJECTION_R" ON "L_LU_REJECTION_REASON"
+  ("REJECTION_REASON_ID" ASC);
+ALTER TABLE "L_LU_REJECTION_REASON" ADD CONSTRAINT
+  "L_PK_LU_TRANSFERS_REJECTION_RE"
```
Comment (MatthewPhinney) [inline src/sql/pg/create.sql]: Should the suffix be `_R` for consistency?

File: src/models/db/transfers.js (hunk 1 above)
Comment (clark800) [inline src/models/db/transfers.js]: Checking for `null` should suffice, I'll make that change

File: src/sql/pg/create.sql (hunk 2 above)
Comment (clark800) [inline src/sql/pg/create.sql]: I don't know, it's just getting truncated to 30 characters

Comment (MatthewPhinney) [inline src/sql/pg/create.sql]: OK, I guess that makes sense. So the convention is to use the table name after the index/constraint name and just truncate at 30 chars.

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**